docker/README.md

```diff
@@ -23,17 +36,11 @@ _Warning: These images are not hardened and shouldn't be used to store real bitc
 
 ```sh
 $ cd bitcoind
-$ docker build --build-arg BITCOIN_VERSION=<version> -t polarlightning/bitcoind:<version> .
+$ docker buildx build  --platform linux/amd64,linux/arm64 --build-arg BITCOIN_VERSION=<version>  -t polarlightning/bitcoind:<version>  --push  .
```

Comment:
There are some double spaces in this command. Can you remove them?

Reply:
removed the spaces, please review